data/hai/hai ships.txt

```diff
@@ -1075,6 +1075,77 @@ ship "Shield Beetle" "Shield Beetle (Tracker)"
 
 
 
+# Flagship of the Unfettered Hai, filled with the spoils of wars against Korath and Wanderers:
+ship "Emperor Beetle"
+	swizzle 6
+	sprite "ship/hai emperor beetle"
+	thumbnail "thumbnail/hai emperor beetle"
+	attributes
+		category "Heavy Warship"
+		"cost" 25100000
+		"shields" 37200
+		"hull" 11780
+		"required crew" 62
+		"bunks" 70
+		"mass" 725
+		"drag" 9.5
+		"heat dissipation" .4
+		"fuel capacity" 500
+		"outfit space" 872
+		"weapon capacity" 333
+		"engine capacity" 150
+		weapon
+			"blast radius" 500
+			"shield damage" 5000
+			"hull damage" 2500
+			"hit force" 3500
+	outfits
+		"Large Heat Shunt" 2
+		"Jump Drive"
+		"Systems Core (Large)"
+		"Systems Core (Medium)"
+		"Quantum Keystone"
+		"Geode Reactor"
+		"White Sun Reactor"
+		`"Bufaer" Atomic Steering`
+		Sunbeam 4
+		"Thunderhead Launcher" 4
+		"Thunderhead Missile" 160
+		"Bullfrog Anti-Missile" 2
+		"Korath Warder"
+		"Thruster (Planetary Class)"
+		"Fuel Processor" 2
+		"Wanderer Ramscoop"
+		"Pulse Rifle" 69
+		"Blade Of Might"
+	engine -45 123
+	engine 45 123
+	gun -22 -103 Sunbeam
+	gun 22 -103 Sunbeam
+	gun -34.5 -68
+	gun 34.5 -68
+	gun -48 -53.5 Sunbeam
+	gun 48 -53.5 Sunbeam
+	gun -61.5 -23
+	gun 61.5 -23
+	turret 0 5 "Korath Warder"
+	turret -55.5 74.5
+	turret 55.5 74.5
+	leak "leak" 35 45
+	leak "flame" 60 95
+	leak "big leak" 50 45
+	explode "tiny explosion" 60
+	explode "small explosion" 51
+	explode "medium explosion" 54
+	explode "large explosion" 53
+	explode "huge explosion" 22
+	"final explode" "final explosion large"
+	description "This ship probably used to be a Shield Beetle, but it has received many modifications, with the obvious intent of survivability, including an enhanced shield matrix. One thing is certain, it is no longer simply meant to escort cargo, this is a proper ship for a warlord."
+	description "	It was found by Unfettered scientists that the cargo bays were more of a vulnerability than anything else for the ship, as it is not meant to be plundering but rather to lead in glorious battle. Removing those as well as surplus bunks has allowed for more space for weapons and defensive improvements, as well as more flexibility for the ship. This came at the cost of mass and agility, but that is a sacrifice the Unfettered are more than willing to make."
+	description "	The Emperor Beetle was built to inspire fear in its enemies, making it known that they face no simple Hai or even Unfettered but rather the foremost military leader of the Unfettered, at the front of those he commands. Some believe it to be foolish to show their leader's ship in such a manner, but that is not how the Unfettered think. Of course, this ship was built to withstand far more punishment than any normal Shield Beetle, while still being able to dish out damage of its own."
```

Comment:
You seem to imply the unfettered are not shallow, foolish, braggarts. They're pragmatic, clever, and obsessed with victory. This is made clear in the Hai Reveal when Teeneep forges an alliance with them. You can think of it as a lore change in 0.9.17a.

Reply:
Well they are clever, this makes for weaknesses for the enemy given how predictible they are, they'll try to go for the throat
Also I like to think the campaign PR implies the things you suggest as well

Reply:
It seems NV's quest against the childish Dumb Nazi vs. Pacifistic Utopian duality of 0.9.12 Hai has spread. I will let her know; she'll be pleased.

Reply:
I have no idea what that means
But I would be happy to change the description if you feel like the strategy behind it isnt well explained enough

Reply:
I'd like to finish play-testing before suggest text changes.